praw/models/reddit/subreddit.py

```diff
@@ -972,6 +992,14 @@ def reports(self, only=None, **generator_kwargs):
         Additional keyword arguments are passed in the initialization of
         :class:`.ListingGenerator`.
 
+        To print the user and mod report reasons in the report queue try:
+
+        .. code:: python
+
+          for report in reddit.subreddit('mod').mod.reports():
```

Comment:
Can we use `item` in stead of report? I like to use item when the results can contain mixed types.